### PR TITLE
[GEOS-9717] Disabling Web Admin Password Autocomplete documentation

### PR DESCRIPTION
**Pull request by imranrajjad**
JIRA : https://osgeo-org.atlassian.net/browse/GEOS-9717

update documentation 

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

For all pull requests:

- [ ] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [ ] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [ ] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [ ] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by travis-ci after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates (screenshots, text)
- [ ] Commits changing the REST API, or any configuration object, should check if the REST API docs (Swagger YAML files and classic documentation) need to be updated.

### CODE REVIEW

File: doc/en/user/source/production/config.rst [hunk 1]
```diff
@@ -76,6 +76,14 @@ In some circumstances, you might want to completely disable the web administrati
 * Set the Java system property GEOSERVER_CONSOLE_DISABLED to true by adding -DGEOSERVER_CONSOLE_DISABLED=true to your container's JVM options
 * Remove all of the gs-web*-.jar files from WEB-INF/lib
 
+Disable the Auto-complete on web administration interface login 
+---------------------------------------------------------------
+
+In a production environment, it is a good practise to turn off Auto Complete on Web Admin login form.
```
Comment (mprins) [inline doc/en/user/source/production/config.rst:82]: I disagree that is is a good practice to try and disable a secure browser behaviour and disable password autocomplete.

for reference also see eg:
- https://security.stackexchange.com/questions/49326/should-websites-be-allowed-to-disable-autocomplete-on-forms-or-fields
- https://bugzilla.mozilla.org/show_bug.cgi?id=425145#c55

Comment (imranrajjad) [reply doc/en/user/source/production/config.rst:82]: @mprins I see, will remove that part from the sentence. Will be glad to if you have any valuable suggestion as to what should the section describe in context what you have been pointed out.

Comment (mprins) [reply doc/en/user/source/production/config.rst:82]: I would use similar wording as the paragraph above for the first sentence, eg.

> In some circumstances, you might want to disable auto-complete on the web admin login form.

and leave opinion out of it.

Comment (imranrajjad) [reply doc/en/user/source/production/config.rst]: done

**Review by mprins — APPROVED** [review]
LGTM

**Comment by aaime** [comment]
There is a documentation build error, but it's unrelated... think it's a PR merged a few days ago. Will fix it separately.

**Comment by aaime** [comment]
Crap, should have squash-merged and instead I've rebase merged... oh well... what's done is done...

**Pull request merged**